pkg/cmd/enclave_secrets.go

```diff
@@ -164,26 +161,48 @@ doppler secrets delete api_key crypto_key`,
 var secretsDownloadCmd = &cobra.Command{
 	Use:   "download <filepath>",
 	Short: "Download a config's .env file",
-	Long: `Save your config's secrets to a .env file. The default filepath is ./doppler.env
+	Long: `Download your config's secrets for later use. Env and JSON format are supported.
+
+Examples:
+
+Save your secrets to /root/ in Env format
+$ doppler enclave secrets download /root/secrets.env
```

Comment:
Can we make json the default? I really think we should move Doppler and it's customers away from this legacy format. I recommend having a `--format=env` flag. This would enable us to support future formats.

Reply:
Will switch json to the default when we add encryption, because then it won't really matter what the format is.